cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/model/Metadata.java

```diff
@@ -0,0 +1,47 @@
+package io.cloudslang.lang.compiler.modeller.model;
+
+import java.util.Map;
+
+/**
+ * User: bancl
+ * Date: 1/11/2016
+ */
+public class Metadata {
+
+    private Map<String, String> inputs;
+    private Map<String, String> outputs;
+    private Map<String, String> results;
+    private String description;
+
+    public Map<String, String> getInputs() {
+        return inputs;
+    }
+
+    public void setInputs(Map<String, String> inputs) {
```

Comment:
I find it easier to read separate setters than a constructor/method with many parameters